src/sonic-config-engine/minigraph.py

```diff
@@ -570,11 +570,22 @@ def parse_dpg(dpg, hname):
                     elif ":" in ipnhaddr:
                         port_nhipv6_map[ipnhfmbr] = ipnhaddr
                 elif ipnh.find(str(QName(ns, "Type"))).text == 'StaticRoute':
-                    prefix = ipnh.find(str(QName(ns, "AssociatedTo"))).text
-                    ifname = ipnh.find(str(QName(ns, "AttachTo"))).text
-                    nexthop = ipnh.find(str(QName(ns, "Address"))).text
-                    advertise = ipnh.find(str(QName(ns, "Advertise"))).text
-                    static_routes[prefix] = {'nexthop': nexthop, 'ifname': ifname, 'advertise': advertise}
+                    prefix = ipnh.find(str(QName(ns, "Address"))).text
+                    ifname = []
+                    nexthop = []
+                    for nexthop_tuple in ipnh.find(str(QName(ns, "AttachTo"))).text.split(";"):
```

Comment:
Do we need to handle backward compatibility? Like, handle a case if old minigraph is loaded?

Reply:
no previous format was defined by us for chassis. Once this PR is merged we will update sonic-mgmt template minigraph_dpg.j2 to use this updated format.  So `deploy-mg` should start using new format.